.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,94 @@
+name: UnitTests
+
+on:
+    - pull_request
+    - push
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php }} + Composer ${{ matrix.dependency }} + Symfony ${{ matrix.symfony }}"
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                include:
+                    # Lowest PHP with the lowest dependencies for every supported lowest major version of the Symfony
+                    -   php: 5.5
+                        dependency: lowest
+                        symfony: '^2.0'
+                    -   php: 5.5
+                        dependency: lowest
+                        symfony: '^3.0'
+                    -   php: 7.1
+                        dependency: lowest
+                        symfony: '^4.0'
+                    -   php: 7.2
+                        dependency: lowest
+                        symfony: '^5.0'
+                    -   php: 8.0
+                        dependency: lowest
+                        symfony: '^6.0'
```

Comment:
```suggestion
```

when i was introducing `lowest` approach to all my projects, I realised that it's enough to do it only for lowest PHP version. Running it on each PHP is overprotective, giving very minimal benefit, if any, and usually means simply a waste of CI cpu

Reply:
Ok i will remove this